packages/mask/src/components/InjectedComponents/ToolboxUnstyled.tsx

```diff
@@ -82,8 +82,56 @@ export interface ToolboxHintProps {
     iconSize?: number
     badgeSize?: number
     mini?: boolean
+    category: 'wallet' | 'application'
 }
 export function ToolboxHintUnstyled(props: ToolboxHintProps) {
+    return props.category === 'wallet' ? <ToolboxHintForWallet {...props} /> : <ToolboxHintForApplication {...props} />
+}
+
+function ToolboxHintForApplication(props: ToolboxHintProps) {
+    const {
+        ListItemButton = MuiListItemButton,
+        Container = 'div',
+        Typography = MuiTypography,
+        iconSize = 24,
+        mini,
+        ListItemText = MuiListItemText,
+    } = props
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    const { openDialog } = useRemoteControlledDialog(WalletMessages.events.ApplicationDialogUpdated)
+    return (
+        <GuideStep step={1} total={4} tip={t('user_guide_tip_1')}>
+            <Container>
+                <ListItemButton onClick={openDialog}>
+                    <img
+                        src={new URL('../../plugins/EVM/assets/maskwallet.png', import.meta.url).toString()}
```

Comment:
please move this image out of the plugin

Reply:
Recommend to use mask icon component.